mon-pix/app/routes/cgu.js

```diff
@@ -0,0 +1,4 @@
+import Route from '@ember/routing/route';
+
+export default class CguRoute extends Route {
```

Comment:
C'est étonnant de voir que ce fichier s'appelle cgu et non pas term-of-services

Reply:
Renommé merci